src/main/java/uk/gov/companieshouse/insolvency/delta/processor/InsolvencyDeltaProcessor.java

```diff
@@ -41,13 +52,18 @@ public InsolvencyDeltaProcessor(InsolvencyDeltaProducer deltaProducer,
     public void processDelta(Message<ChsDelta> chsDelta) {
         try {
             MessageHeaders headers = chsDelta.getHeaders();
+            final String logContext = chsDelta.getPayload().getContextId();
+            final Map<String, Object> logMap = new HashMap<>();
             final String receivedTopic =
                     Objects.requireNonNull(headers.get(KafkaHeaders.RECEIVED_TOPIC)).toString();
             final ChsDelta payload = chsDelta.getPayload();
 
             ObjectMapper mapper = new ObjectMapper();
             InsolvencyDelta insolvencyDelta = mapper.readValue(payload.getData(),
                     InsolvencyDelta.class);
+            Insolvency insolvency = insolvencyDelta.getInsolvency().get(0);
+            InternalCompanyInsolvency internalCompanyInsolvency = transformer.transform(insolvency);
+            final String companyNumber = insolvency.getCompanyNumber();
```

Comment:
Let's keep those where they were further down (with the addition of the company number) - the code comment specifically applies to them.

Reply:
Addressed now